tailcall-macros/src/document_definition.rs

```diff
@@ -0,0 +1,109 @@
+extern crate proc_macro;
+use proc_macro::TokenStream;
+use quote::quote;
+use syn::{parse_macro_input, DeriveInput};
+
+#[derive(Default)]
+struct DirectiveDefinitionAttr {
+    is_repeatable: bool,
+    is_lowercase_name: bool,
+    locations: Option<String>,
+}
+
+fn get_directive_definition_attr(input: &DeriveInput) -> syn::Result<DirectiveDefinitionAttr> {
+    let mut directve_definition_attr: DirectiveDefinitionAttr = Default::default();
```

Comment:
```suggestion
    let mut directive_definition_attr: DirectiveDefinitionAttr = Default::default();
```

Reply:
I'm so sorry for those typos, I should've been more careful. 

I ran my spell checker and all of them should be fixed now, thank you for pointing it out